tests/ballerina-test/src/test/resources/testng.xml

```diff
@@ -93,6 +93,7 @@ under the License.
 			<class name="org.ballerinalang.test.types.map.BMapValueTest" />
 			<class name="org.ballerinalang.test.types.map.MapAccessExprTest" />
 			<class name="org.ballerinalang.test.types.map.MapInitializerExprTest" />
+		        <class name="org.ballerinalang.test.net.websocket.WebSocketCompilerPluginTest" />
```

Comment:
Why not add the whole package as other tests? Judging by the `TODO` comment on this, this list seems to be a temporary one